### ISSUE TITLE
Crash After Boot

### ISSUE THREAD

**Issue by Codemonkey03** [xerpi/vita-udcd-uvc#43]
Using Vita 2000 Slim firmware 3.60 enso. Tried the normal udcd plugin and the no-LCD plugin. Whenever the system reboots after installing it will crash after the warning message. I've tried moving the line for the plugin inside the config file in my ur0 tai folder as others have suggested but no matter where I put it nothing changes.

Any suggestions are all would be appreciated. I'd love to have this up and running; already got the Vita Dock on my pi setup but the wall I hit is trying to get the plugin to work.

**Comment by xerpi** [comment]
Check https://github.com/xerpi/vita-udcd-uvc/issues/34#issuecomment-583938153

**Comment by xerpi** [comment]
Looks like you fixed it by adding more delay? (https://github.com/xerpi/vita-udcd-uvc/issues/34#issuecomment-583938153)

**Comment by Codemonkey03** [comment]
> Looks like you fixed it by adding more delay? ([#34 (comment)](https://github.com/xerpi/vita-udcd-uvc/issues/34#issuecomment-583938153))

Yes I found that other post and used that skprx with the 10 second delay and it actually booted and it appeared to work (almost). I'm using this plugin with Vita dock and when I plug into the dock the very first time the Vita is displayed and works fine. If I unplug it the vita and plug it back in the system will freeze for several seconds before resuming normal operation and nothing will be output to the dock. The vita and dock both need to be power cycled at that point to reconnect and have it show up on the screen again and even this method has been unreliable to me. 

But ultimately it seems that the delay was the biggest issue keeping my Vita from booting at all. I have quite a few plugins and startup usually takes 10-12 seconds from boot to live menu. I have a hard time finding reference for normal operation with vita dock so I don't know how the plugin is supposed to interface with it normally but it seems like the plugin is crashing or stops working after the first connect.

I hate to ask but is it possible for you to make the edits to the delay for me on the no_led version of the skrpx? I'm really unfamiliar with all this and although I can mess around in the config txt files just fine anything more and I'm lost. 10 works but but I'd be curious to see what happens with 15 and 20 seconds in delay.

**Comment by xerpi** [comment]
Install [vitasdk](https://vitasdk.org/), then edit this line: https://github.com/xerpi/vita-udcd-uvc/blob/master/main.c#L815
And run the `gen_builds.sh` script.
